apps/hmi/base/kustomization.yaml

```diff
@@ -3,5 +3,6 @@ kind: Kustomization
 resources:
   - ../../base
   - ../hmi-rota-dtu/hmi-rota-dtu.yaml
+  - ../hmi-libra-dtu/hmi-libra-dtu.yaml
```

Comment:
This add it to all environments? Are u sure u want to add it to all envs? Do have enough approvals?

Reply:
Yes. We want it in all except prod for now.

Reply:
This might add it to prod as well if hmc is in prod. see https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#add-application-to-all-environments

Reply:
HMI namespace isn't in prod. Have removed new libra config from here, will add back when we go into prod.